npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser.rb

```diff
@@ -110,7 +110,8 @@ def lockfiles
         {
           npm: package_lock || shrinkwrap,
           yarn: yarn_lock,
-          pnpm: pnpm_lock
+          pnpm: pnpm_lock,
+          bun: bun_lock
```

Comment:
I wish people stop appending items to the list, if that involves changing previous line due trailing comma.

1. solutions: add the item anywhere than last, then the dif would not involve trailing comma
2. solution2: reformat code first to always have trailing comma in the array